src/meta/name.rs

```diff
@@ -111,7 +111,7 @@ impl Name {
     }
 
     pub fn extension(&self) -> Option<&str> {
-        self.extension.as_ref().map(|string| string.as_str())
+        self.extension.as_deref()
```

Comment:
This seems to fail on 1.33

Reply:
reverted. This commit comes from the warning provided by clippy. The Rust compiler included in the github action should also be downgraded to 1.33.

Reply:
Hmm, let me look into it.

Reply:
Hey, sorry for the late reply. I am pretty busy with work recently.
I think we will the min supported version to 1.34 as lscolors already did it https://github.com/sharkdp/lscolors/commit/936e518d6c3a26a1ddc812d8d40e542b916888b1 

Could you revert to the code and just bump the rust version in travis to 1.34
https://github.com/Peltoche/lsd/blob/88e4133e16eee821cb3c9cc2c84aa7f995e22ed2/.travis.yml#L44

Reply:
as_deref() is supported since 1.40. it doesn't need to be revert. The Trevis Rust version bumpup will be conducted as a separate pull request.

Reply:
Oh, my bad. I was hoping we could get rid o f the clippy warnings.